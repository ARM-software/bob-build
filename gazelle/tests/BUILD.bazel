load("@bazel_gazelle//:def.bzl", "gazelle_generation_test")

[gazelle_generation_test(
    name = file[0:-len("/WORKSPACE")],
    size = "small",
    gazelle_binary = "//:gazelle_bin",
    test_data = glob(
        include = [file[0:-len("/WORKSPACE")] + "/**"],
    ),
) for file in glob(["**/WORKSPACE"])]
