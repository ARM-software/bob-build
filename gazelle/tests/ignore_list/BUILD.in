# gazelle:bob_root
# gazelle:bob_ignore subdir/subdir_nested
# gazelle:bob_ignore subdir_two
