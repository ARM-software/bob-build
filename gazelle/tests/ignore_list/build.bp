// Main build file

bob_filegroup {
    name: "should_include",
    srcs: ["*.c"],
}
