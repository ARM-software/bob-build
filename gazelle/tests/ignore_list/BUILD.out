# gazelle:bob_root
# gazelle:exclude subdir/subdir_nested
# gazelle:exclude subdir_two

filegroup(
    name = "should_include",
    srcs = glob(["*.c"]),
)
